dev/image/Dockerfile

```diff
@@ -30,6 +33,9 @@ RUN mkdir -p /usr/local/kubernetes/ && \
 
 ENV PATH=$PATH:/usr/local/kubernetes/cluster/:/usr/local/kubernetes/client/bin/
 
+RUN sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 15CF4D18AF4F7421 && \
```

Comment:
Hey @princerachit can you try removing these, and rebuilding the dev image with the latest `europe-docker.pkg.dev/gitpod-artifacts/docker-dev/workspace-base-images:full`? It should work better no, and no longer get a NO_PUBKEY error.